src/modules/tracker.js

```diff
@@ -398,7 +398,8 @@ class Tracker {
    * @param {object} [networkParameters] - Parameters relevant to the network request
    * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
    * @returns {(true|Error)}
-   * @description User selected (clicked, or navigated to via keyboard) a result that appeared within autocomplete
+   * @description User selected (clicked, or navigated to via keyboard) a result that appeared
+   * within autocomplete (Search Suggestions, Products, or a custom section)
```

Comment:
Can we add examples of custom section? something like (or a custom section eg. Brands, Categories)

Reply:
Good idea. Fixed [cabd50f](https://github.com/Constructor-io/constructorio-node/pull/210/commits/cabd50f7d0473334077b1fd058197de88356bb6a)